src/App.tsx

```diff
@@ -1,22 +1,26 @@
-import React, { useEffect, useState } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
+import Footer from './components/Footer/Footer';
 import Header from './components/Header/Header';
 import Main from './components/Main/Main';
+import Modal from './components/Modal/Modal';
+import ModalManager from './components/ModalManager/ModalManager';
+import Search from './components/Search/Search';
 import filmPosts from './mock';
 import { Post } from './types/interfaces';
-import Footer from './components/Footer/Footer';
+import { SortingValue } from './types/types';
 import { filterPostsByGenre, filterPostsByName } from './utils/filterPosts';
 import sortPosts from './utils/sortPost';
-import { SortingValue } from './types/types';
+import { AppContext } from './Context';
```

Comment:
Don't forget to remove unused imports 
Also it's just a recommendation, but take a look on this extensions `"plugin:import/errors", "plugin:import/warnings"` for eslint. It helps to automatically solve unused imports or warn you if it exists